drake/systems/plants/@Manipulator/Manipulator.m

```diff
@@ -54,7 +54,7 @@
         % want to raise for this method, stating that not all outputs were
         % assigned.  (since I can't write dxdot anymore)
         [H,C,B,dH,dC,dB] = obj.manipulatorDynamics(q,v);
-        Hinv = inv(H);
+        Hinv = pinv(H);
```

Comment:
During the gradient descent on parameter values, it comes up with Hinv that are not well conditioned.  I changed it because I found that pinv was more stable and faster.  But yes, I agree it is a big change.  I will change it back.